kernel/common/enumerable.rb

```diff
@@ -212,6 +212,12 @@ def slice_when(&block)
     end
   end
 
+  def chunk_while(&block)
+    raise ArgumentError, "wrong number of arguments (0 for 1)" unless block_given?
```

Comment:
Ah, I see now why the block is required. So yeah, we should fix the error message.